dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/pom.xml

```diff
@@ -141,5 +141,10 @@
             <groupId>com.zaxxer</groupId>
             <artifactId>HikariCP</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>druid</artifactId>
+        </dependency>
     </dependencies>
```

Comment:
Why are you introducing a new database connection pool? we already have `HikariCP`.

Reply:
HikariCP does not support Dameng, druid supports Dameng

Reply:
I don't understand, it's just a data source connector management, why doesn't it support dm, can you explain why?

Reply:
thanks，I just tested the DaMeng database with HikariCP，HikariCP supports DaMeng